recipes/glib/all/conanfile.py

```diff
@@ -79,10 +79,10 @@ def requirements(self):
         if self.options.get_safe("with_mount"):
             self.requires("libmount/2.36.2")
         if self.options.get_safe("with_selinux"):
-            self.requires("libselinux/3.2")
+            self.requires("libselinux/3.3")
         if self.settings.os != "Linux":
             # for Linux, gettext is provided by libc
-            self.requires("libgettext/0.20.1")
+            self.requires("gettext/0.21")
```

Comment:
```suggestion
            self.requires("libgettext/0.20.1")
```
gettext is package only an executable, here we need the library, so libgettext package